plugin/src/main/java/com/denizenscript/denizen/utilities/Utilities.java

```diff
@@ -645,4 +649,41 @@ public static boolean requireEnumlike(Mechanism mechanism, Class<?> type) {
         }
         return true;
     }
+
+    public static ServerLinks replaceServerLinks(ServerLinks serverLinks, ListTag list, TagContext context) {
+        serverLinks.getLinks().forEach(serverLinks::removeLink);
+        return fillServerLinks(serverLinks, list, context);
+    }
+
+    public static ServerLinks fillServerLinks(ServerLinks serverLinks, ListTag list, TagContext context) {
+        for (MapTag map : list.filter(MapTag.class, context)) {
+            if (!map.containsKey("uri")) {
+                Debug.echoError("Invalid ServerLink map: missing 'uri' key!");

```

Comment:
`ServerLink` is the internal Java name, for Denizen errors can just use something like `Invalid server links map`